### ISSUE TITLE
Performance benchmarks

### ISSUE THREAD

**Issue by ScriptedAlchemy** [dai-shi/react-hooks-worker#11]
What's the perf like on cold-starting workers. Your Fib example shows visibility faster calculations compared to normal JS implementations. 

Would love to see a perf benchmark - very interested in using this in production

**Comment by dai-shi** [comment]
Yeah, it would be interesting. I'll think about that. My priority is designing new API #10 though.
Do you have any specific use cases? I mean, the fib example is artificial and it may or may not be suitable for benchmarks.

**Comment by dai-shi** [comment]
Although not specifically related this project, I did benchmarks with redux-in-worker.
[The current result](https://github.com/dai-shi/redux-in-worker/issues/1#issuecomment-522826500) shows an increase in "script bootup time," which might help understand the cost of starting a worker.
